### PR TITLE
Incorporates edits from Jason

### PR DESCRIPTION
**Pull request by arokem**
I am not sure I got everything. I will give this another pass tomorrow.

### CODE REVIEW

File: reports/manuscript/sections/conclusion.tex [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
+well in predicting both continuous variables, such as biological age, as well as
+categorical variables, such as whether a person is a patient or a healthy control.
+In both of these cases, SGL out-performs previous algorithms that have been
+developed for these tasks \cite{sarica2017corticospinal, Richard2018-ux}. The
+nested cross-validation approach used to fit the model and make both predictions
+and inferences from it guards against overfitting and tunes the degree of
+sparseness required by the algorithm. This means that both phenomena that are
+locally focused in a particular anatomical location and a particular property of
+diffusion (e.g., FA in the CST), as well as widely distributed phenomena can be
+accurately captured.
```
Comment (richford) [inline reports/manuscript/sections/conclusion.tex]: Recommend changing this last sentence to active voice with itemization:
"Thus, SGL can accurately describe both (a) phenomena that are locally confined to a particular anatomical location or diffusion property (e.g., FA in the CST), as well as (b) phenomena that are widely distributed amongst brain regions and measured diffusion properties."

or something like that.

**Comment by richford** [comment]
Made one small suggestion but otherwise looks good to me! I'm assuming since it's labelled WIP that you want me to hold off on merging. I think you're traveling today anyway so no rush. Safe travels!

File: reports/manuscript/sections/conclusion.tex (hunk 1 above)
Comment (arokem) [reply reports/manuscript/sections/conclusion.tex]: Yes! Addressed in 53d9b86

**Comment by arokem** [comment]
I'm going to go ahead and merge this, so we can share the current master with others for their comments.

**Pull request merged**